src/error_handling_helpers.h

```diff
@@ -36,6 +36,7 @@
 #endif
 #include <assert.h>
 #include <stdlib.h>
+#define __STDC_WANT_LIB_EXT1__ 1
```

Comment:
This is already available a couple of lines earlier, so I don't think we need this here.

```suggestion
```

Reply:
@clalancette I like your idea